harness/determined/_core/_train.py

```diff
@@ -133,6 +158,14 @@ def report_validation_metrics(
             self._tbd_mgr.sync()
 
     def report_early_exit(self, reason: EarlyExitReason) -> None:
+        """
+        Report an early exit reason to the Determined master.
+
+        Presently, the only meaningful value to report is EarlyExitReason.INVALID_HP, which is
```

Comment:
"Currently", not "Presently".
Class names, method names, argument names, and literal values should have syntax highlighting. Probably, monospace, although I like to use italics for argument names.